### PR TITLE
Cleanup and addition of private network writeup

### PR DESCRIPTION
**Pull request by fr1t2**

### CODE REVIEW

File: _docs/developers/privateNetwork.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
-# Setting up a QRL private network
-This documentation assumes that you have already followed the QRL node installation instructions as mentioned in this link  https://docs.theqrl.org/node/QRLnode/
+---
+title: QRL Private Network
+categories: developers
+tags: developers
+---
 
-### 1. Configuring config.yml
+Setting up a QRL private network will allow a developer the ability to test out transacting on the QRL network without the risk of testing with real QRL. This is intedned to help the development and advancement of external systems.
```
[the remark this answers is not in the document]
Comment (fr1t2) [reply _docs/developers/privateNetwork.md]: fixed

File: _docs/developers/privateNetwork.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ categories: developers
 tags: developers
 ---
 
-Setting up a QRL private network will allow a developer the ability to test out transacting on the QRL network without the risk of testing with real QRL. This is intedned to help the development and advancement of external systems.
+Setting up a QRL private network will allow a developer the ability to test out transacting on the QRL network without the risk of testing with real QRL. This is intendned to help the development and advancement of external systems.
```
Comment (cyyber) [inline _docs/developers/privateNetwork.md]: Typo "intendned" should be "intended"

Comment (fr1t2) [reply _docs/developers/privateNetwork.md]: Typo's typo fixed. Spell check enabled.

File: _docs/developers/privateNetwork.md [hunk 3]
```diff
@@ -1,9 +1,18 @@
-# Setting up a QRL private network
-This documentation assumes that you have already followed the QRL node installation instructions as mentioned in this link  https://docs.theqrl.org/node/QRLnode/
+---
+title: QRL Private Network
+categories: developers
+tags: developers
+---
 
-### 1. Configuring config.yml
+Setting up a QRL private network will allow a developer the ability to test out transacting on the QRL network without the risk of testing with real QRL. This is intended to help the development and advancement of external systems.
```
[the remark this answers is not in the document]
Comment (fr1t2) [reply _docs/developers/privateNetwork.md]: Updated to include wording.

**Pull request merged**